Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN pip install --no-cache-dir --upgrade --prefix="/install" -r requirements.txt
 FROM python:3.10-slim-bookworm as final
 
 RUN apt-get update --fix-missing && apt-get install -y --fix-missing \
+    bash \
```

Comment:
Why is this needed?

<img width="711" alt="image" src="https://github.com/chroma-core/chroma/assets/1157440/d483bf66-93f9-46af-8224-3b55e62cc4e0">

Reply:
removed